src/mca/rmaps/base/rmaps_base_ranking.c

```diff
@@ -152,7 +152,7 @@ static int rank_span(prrte_job_t *jdata,
                         }
                         prrte_output_verbose(5, prrte_rmaps_base_framework.framework_output,
                                             "mca:rmaps:rank_span: assigning vpid %s", PRRTE_VPID_PRINT(vpid));
-                        proc->name.vpid = vpid++;
+                        proc->name.vpid = proc -> rank = vpid++;
```

Comment:
We generally separate these into two lines instead of doing the "laddered" equates

Reply:
Will change.